app/controllers/miq_policy_controller/policies.rb

```diff
@@ -118,15 +118,13 @@ def policy_delete
     # showing 1 policy, delete it
     pol = MiqPolicy.find_by_id(params[:id])
     if params[:id].nil? || pol.nil?
-      add_flash(_("%{models} no longer exists") % {:models => ui_lookup(:model => "MiqPolicy")},
-                :error)
+      add_flash(_("Policy no longer exists"), :error)
     else
       policies.push(params[:id])
       self.x_node = @new_policy_node = "xx-#{pol.mode.downcase}_xx-#{pol.mode.downcase}-#{pol.towhat.downcase}"
     end
     process_policies(policies, "destroy") unless policies.empty?
-    add_flash(_("The selected %{models} was deleted") %
-      {:models => ui_lookup(:models => "MiqPolicy")}) if @flash_array.nil?
+    add_flash(_("The selected Policy was deleted")) if @flash_array.nil?
```

Comment:
This is deliberate plural -> singular correction.  The action is deleting 1 (currently selected) policy.

(not seeing this flash in practice)